lib/matplotlib/__init__.py

```diff
@@ -1251,21 +1251,24 @@ def interactive(b):
     """
     Set interactive mode to boolean b.
```

Comment:
Something like "Set whether to draw after every plotting command (such as `~.Axes.xlabel`)." fits in a single line (replacing the entire docstring).
`